package.json

```diff
@@ -5,14 +5,14 @@
   "main": "index.js",
   "dependencies": {
```

Comment:
There're several packages that are no in use anymore, here's a list and could you check and delete them: 
d3(already have partial packages so we don't need this whole package anymore) , fast-csv, js-file-download, material-design-icons, material-ui-icons, multisort, npm, react-form, react-highlight, react-motion, react-remarkable, svg-fill-loader

Reply:
I used `depcheck` and remove the ones it showed - the ones you listed + cssnano and polished